linera-rpc/src/grpc/transport.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::NodeOptions;
+
+#[derive(Clone, Debug, Default)]
+pub struct Options {
+    pub connect_timeout: Option<std::time::Duration>,
+    pub timeout: Option<std::time::Duration>,
+}
+
+impl From<&'_ NodeOptions> for Options {
+    fn from(node_options: &NodeOptions) -> Self {
+        Self {
+            connect_timeout: Some(node_options.send_timeout),
+            timeout: Some(node_options.recv_timeout),
+        }
+    }
+}
+
+#[cfg(web)]
+mod implementation {
+    use super::Options;
+
+    pub use tonic_web_wasm_client::{Client as Channel, Error};
+
+    pub fn create_channel(address: String, _options: &Options) -> Result<Channel, super::Error> {
```

Comment:
Minor: maybe use just `Error` to make it easier to find the import?

```suggestion
    pub fn create_channel(address: String, _options: &Options) -> Result<Channel, Error> {
```

Reply:
I don't think I fully understand this comment… how would removing the information about where the name is make it easier to find?